LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/Model/DBPUICommunicationModel.swift

```diff
@@ -135,7 +146,79 @@ struct DBPUIDataBrokerProfileMatch: Codable {
     let addresses: [DBPUIUserProfileAddress]
     let alternativeNames: [String]
     let relatives: [String]
-    let date: Double? // Used in some methods to set the removedDate or found date
+    let foundDate: Double
+    let optOutSubmittedDate: Double?
+    let estimatedRemovalDate: Double?
+    let removedDate: Double?
+    let hasMatchingRecordOnParentBroker: Bool
+}
+
+extension DBPUIDataBrokerProfileMatch {
+    init(optOutJobData: OptOutJobData,
```

Comment:
I don't really like how long this construct got, but didn't have any nice ideas to simplify it, so taking suggestions!

Reply:
Tried rewriting this a different way, but it was mostly just different, not necessarily better. However, leaving one suggestion.